Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -1893,7 +1950,14 @@ def RequestData(self, request, inInfo, outInfo):
                 sa.InsertNextValue(el)
             reader.SetFileNames(sa)
             reader.Update()
-            outData.ShallowCopy(reader.GetOutput())
+            data = reader.GetOutput()
+            if VTK_MAJOR_VERSION == 8:
+                # In this case we have to flip along the y axis
+                data = Converter.vtk2numpy(reader.GetOutput())
+                data = np.flip(data, axis=1)
+                data = Converter.numpy2vtkImage(data)
```

Comment:
This is not necessary now, right?

Reply:
not sure how I missed this one

Reply:
or how it passed the tests becuase VTK_MAJOR_VERSION doesn't exist.......